### ISSUE TITLE
Non-display information about channel quality settings

### ISSUE THREAD

**Issue by djgraco** [BearWare/TeamTalk5#1084]
Hello,
On TeamTalk 5 QT version 5.8.2.5053 there is a problem with displaying information about some channels.
I don't know what it depends on, but even if I have administrator rights, I can't see the codec, bitrate, sample rate and channels settings in the shift+f8 channel edit window. Variable bitrate and frame settings are visible.
This does not happen with all channels, on some channels, after pressing shift + f8 I can edit the codec, bitrate, channels and sample rate properties as normal.

**Comment by djgraco** [comment]
I think I've figured out how to trigger an error.
1. Create a new channel without touching any sound quality settings. Once created, hover over it and press shift + f8. We won't see the codec, bitrate, sample rate and channel settings in the window.

**Comment by bear101** [comment]
I don't see a "Shift+F8" shortcut anywhere

**Comment by djgraco** [comment]
Oh, sorry, I'an maked a mistake. I meant shift+f7

**Comment by bear101** [comment]
Is it Windows, macOS or Ubuntu?

**Comment by bear101** [comment]
Also is it a screen reader not announcing the items? If, so, which one?

**Comment by djgraco** [comment]
It is Windows 10 Pro, 64 bit. Client is TeamTalk 5 QT. Items bitrate, channels, codec type are a wisible with NVDA, but they empty and not settable.

**Comment by bear101** [comment]
You cannot modify audio codec settings if there's people in a channel. The channel must be empty in order to modify settings.

**Comment by djgraco** [comment]
I know about it.
I'll explain it differently.
As we do the steps I gave in post 2, after pressing shift + f7 the information about the codec, bitrate, channel, type of transmission are empty, which means that you can not even see the quality of the channel. This does not apply to all channels, e.g. with other channels, such settings are normally visible.
I still have to check if it really depends on whether the person is on the channel or not.

**Comment by bear101** [comment]
@Oreonan Is this a Qt Framework issue that is already known?

**Comment by Oreonan** [comment]
I'm not really sure that is an accessibility issue. I think some informations are really empties when you tried to edit a channel.
I tested with a channel containing peoples, the channel is the root channel, and I'm not admin on this server.
If necessary I can try to confirm if this informations are really empties or if it's finally an accessibility issue.
